### PR TITLE
Change on what exactly must be secured in a presentation

### PR DESCRIPTION
**Pull request by iherman**
This is to answer to the issue raised in #1512

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1515.html" title="Last updated on Jul 2, 2024, 9:25 AM UTC (501eb24)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1515/5dd94d3...501eb24.html" title="Last updated on Jul 2, 2024, 9:25 AM UTC (501eb24)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4266,9 +4266,10 @@ <h3>Securing Mechanism Specifications</h3>
 [=default graph=].
```
Comment (dlongley) [inline index.html:4266]: I can't make a suggestion here, but I believe this should not be a statement about a "property" but about a securing mechanism, and it should be simplified to "The securing mechanism MUST secure all graphs in a [=verifiable credential=]." There's a minor technicality in that, a proof value itself (in a proof graph itself) cannot be "self-protected", but maybe that's obvious.

Similarly, the above "The property MUST define" seems more like it should say "The securing mechanism MUST define". This list should perhaps be about the requirements for the property *and* the new securing mechanism.

File: index.html [hunk 2]
```diff
@@ -4266,9 +4266,10 @@ <h3>Securing Mechanism Specifications</h3>
...
           </li>
           <li>
-In a [=verifiable presentation=], the property MUST secure the
-[=default graph=] of the [=presentation=] as well as every [=proof graph=]
-related to each [=verifiable credential=] in the [=presentation=].
+In a [=verifiable presentation=], the property MUST secure the 
+[=default graph=] of the [=presentation=], and each 
+[=verifiable credential=], as well as its related [=proof graph=], in the 
+[=presentation=].
```
Comment (dlongley) [inline index.html:4272]: Hmm, something more like this?

```suggestion
In a [=verifiable presentation=], the property MUST secure all
graphs in the [=presentation=], including each 
[=verifiable credential=] and any of its related [=proof graphs=].
```

**Review by decentralgabe — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html:4266]: I wonder whether —
```
Securing mechanism specifications that create new types of [=embedded proofs=] 
MUST specify a [=property=] for securing
```

— should become the following (which implies further changes to the bullet list, and possibly more) — 
```
Securing mechanism specifications that create new types of [=embedded proofs=]
MUST specify an object for securing both [=verifiable credentials=] and
[=verifiable presentations=].
```
— (should `object` be `[=object=]`?) with appropriate bullet adjustments, such as — 
```
The object MUST provide values for all [=properties] used by the [=proof graph=].
```

File: index.html (hunk 2 above)
Comment (TallTed) [reply index.html:4272]: Along the lines of my above, including @dlongley's suggestion --
```suggestion
In a [=verifiable presentation=], the object MUST secure all graphs in
the [=presentation=] not referred to by the object itself, including
each [=verifiable credential=] and any of its related [=proof graphs=].
```

**Review by longpd — APPROVED** [review]
With dlongley's amendments made by TallTed I support PR #1515

File: index.html (hunk 2 above)
Comment (dlongley) [reply index.html:4272]: @TallTed, I think this should probably say: "the object MUST secure all graphs in
the [=presentation=] not referred to by the property,", right?

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html:4266]: I not against making this change, but that is more complex change than what this PR proposes to do. I would prefer to finalize this PR first and, if needed, do that in a separate PR.

File: index.html (hunk 2 above)
Comment (iherman) [reply index.html]: See my comment above. I'd prefer to close this PR and look at the overall change in a separate PR.

**Comment by iherman** [comment]
I have merged the proposal of @dlongley but I am not absolutely sure this is 1005 correct. Let us take the hypothetical situation as follows.

- I use a securing mechanism using `proof` from DI with one of the cryptosuites.
- I ***also*** use, in the same VP (or VC) a securing mechanism that defines the property `signature`.

Question: does `proof` have to also secure the proof graph defined by `signature`? In my original text, which explicitly lists what has to be secured, the answer is 'no'. In the text proposed by @dlongley the answer is, actually, 'yes'? I believe the intention is 'no', because the goal is to secure the core information in the VC/VP, but that requires an explicit listing of _what_ is secured...

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html:4266]: B.t.w., @TallTed, the (RDF) "object" term is not used in the core VCDM terminology, only "value".

**Review by msporny — APPROVED** [review]

**Comment by iherman** [comment]
We should close this PR without merge, and concentrate on teh more comprehensive #1519 instead.

**Comment by iherman** [comment]
The discussion has clearly moved to #1519, closing this PR without further action, to avoid misunderstandings.

**Pull request closed without merging**